### DISCUSSION TITLE
termux no one games works

### DISCUSSION THREAD

**Discussion by Plhorror** [community/community#78531] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi,
in termux no one games works i tested all drivers but nothing works!

i tested resident evil 5 game won't install.
i tested road redemption game launch but crash after 10 seconds.
i tested carmageddon max damage but crash just before playing.

i tested this 3 games and they are not working on winlator to

i have sd870 with adreno 650 

whats wrong?
I forgot something

**Comment by Plhorror** [comment]
any ideas?

**Comment by George2514** [comment]
Suika Game plays similarly to Sega's Puyo Puyo Tetris, [Suika Online Games](https://suika-games.com/) in which you have to line up matching fruits to make them disappear before the basket overflows.

**Comment by Plhorror** [comment]
?????
